pkg/controller/directvolumemigration/rsync.go

```diff
@@ -74,12 +74,10 @@ func (t *Task) areRsyncTransferPodsRunning() (bool, error) {
 	}
 
 	pvcMap := t.getPVCNamespaceMap()
+	dvmLabels := t.getDVMLabels()
+	dvmLabels["purpose"] = Rsync
```

Comment:
Suggestion: maybe rename the `Rsync` const to `RsyncPurpose` or something that gives more context of what is contained within, since this is a pkg level var.